### PR TITLE
build: Add dependabot config for automatic dependency updates.

### PR DESCRIPTION
**Pull request by klkucaj**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "pub"
+    directory: "/"
```
Comment (SandPod) [inline .github/dependabot.yml:4]: Question: Is this required?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    schedule:
+      interval: "daily"
+    versioning-strategy: "widen"
+    # Limit open pull requests to 1 to avoid multiple PRs for the same dependency.
+    open-pull-requests-limit: 1
```
Comment (SandPod) [inline .github/dependabot.yml:9]: Suggestion: Set this to 5, it is OK to have multiple requests open here.

Update the comment as well.

File: .github/dependabot.yml (hunk 1 above)
Comment (klkucaj) [reply .github/dependabot.yml:4]: yes it is required https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

Screenshot 2024-12-13 at 12 15 05 PM

**Review by SandPod — APPROVED** [review]
LGTM! 👍

**Pull request merged**